tests/components/aprs/test_device_tracker.py

```diff
@@ -16,6 +17,13 @@
 TEST_PASSWORD = "testpass"
 
 
+@pytest.fixture(name="mock_ais")
+def mock_ais():
```

Comment:
Can we add type hints here and on all the tests?
```suggestion
def mock_ais() -> Generator[MagicMock, None, None]
```

Reply:
While thats a good idea, adding type hints seems beyond the scope of getting the tests working for python 3.11